product_supplierinfo_for_customer_picking/models/stock_move.py

```diff
@@ -23,9 +24,21 @@ def _compute_product_customer_code(self):
                     )
                 )
                 product_customer_code = customer.product_code
+                product_customer_name = customer.product_name
             move.product_customer_code = product_customer_code
+            move.product_customer_name = product_customer_name
+            product = move.product_id.with_context(
+                partner_id=move.picking_id.partner_id.id
+            )
+            description = product.name_get()
+            if description:
```

Comment:
Any chance to get an empty description by calling `name_get`?
Also, wouldn't be better to call `product._get_description(self.picking_type_id)`?

Reply:
Yes, if there is no specific to customer name.
_get_description will get you standard product name. Not a customer specific.
Commit name was shortened.